vital/vital_config.h.in

```diff
@@ -10,34 +10,35 @@
  */
 
 ///////////////////////////////////////////////////////////////////////////////////////////////
-////// Do not edit this file. It is generated by CMake. All local changes will be lost. ///////
+////// Do not edit this file. It is generated by CMake. All local changes will
+/// be lost. ///////
 ///////////////////////////////////////////////////////////////////////////////////////////////
 
 #ifndef VITAL_VITAL_CONFIG_H
 #define VITAL_VITAL_CONFIG_H
 
 // Support macros.
-#if defined(_WIN32) || defined(_WIN64)
+#if defined( _WIN32 ) || defined( _WIN64 )
 
 // TODO use C++17 attributes if available
 
 # define VITAL_MUST_USE_RESULT  /* unsupported */
 # define VITAL_UNUSED           /* unsupported */
-# define VITAL_SYSTEM_WIN_API (1)
+# define VITAL_SYSTEM_WIN_API ( 1 )
 
-#elif defined(__GNUC__)
+#elif defined( __GNUC__ )
 
-# define VITAL_MUST_USE_RESULT __attribute__((__warn_unused_result__))
-# define VITAL_UNUSED __attribute__((__unused__))
-# define VITAL_SYSTEM_WIN_API (0)
+# define VITAL_MUST_USE_RESULT __attribute__( ( __warn_unused_result__ ) )
+# define VITAL_UNUSED __attribute__( ( __unused__ ) )
```

Comment:
Uh... this seems dodgy, but apparently doesn't actually make the compilers happy, so I guess we can ignore it for now.